openstack/placement/templates/stamp-alembic-job.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if and (not .Values.mariadb.enabled) .Values.api_db.initial_stamp -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "placement-stamp-alembic"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ default "unknown" .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+    metadata:
+      name: "placement-stamp-alembic"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: stamp
+        image: {{ required ".Values.global.dockerHubMirrorAlternateRegion is missing" .Values.global.dockerHubMirrorAlternateRegion }}/{{ .Values.mariadb.image | default "library/mariadb:10.5.12" }}
+        command:
+        - /usr/bin/env
+        - bash
+        - -c
+        - |
+          mariadb -v -hnova-api-mariadb -u{{ .Values.api_db.user }} --password='{{ .Values.api_db.password }}' '{{ .Values.api_db.name }}' -e '
```

Comment:
`nova-api-mariadb` is hardcoded, because we only need to run this once and don't know how/when if ever we split off placement from there, right?

Reply:
Exactly, it is because it is a one-time thing. Any new deployment will (hopefully) be on a newer version of placement anyway.